docs/source/tutorials/chat.rst

```diff
@@ -294,6 +295,7 @@ object.
             # For local csv files, we specify "csv" as the source, just like in
             # load_dataset
             source="csv",
+            split="train",
```

Comment:
also would be good to add a comment above this line saying specifying the default split of "train" is required for local files

also if you don't mind `split="train"` to the examples here as well: https://github.com/pytorch/torchtune/blob/dc742677d326b4536fb91f465e3d5499f389a2d7/docs/source/tutorials/datasets.rst?plain=1#L417

Reply:
Done :heavy_check_mark: